frontend/src/pages/auth/DeregisterButton.tsx

```diff
@@ -24,15 +24,16 @@ const DeregisterButton: React.FC = () => {
         });
       });
   };
+
   return (
-    <ConfirmationDialog
-      dialogHeader={'Delete Account'}
-      dialogBody={'Are you sure you want to delete your account? This action is irreversible!'}
-      mainButtonLabel={'Delete Account'}
-      rightButtonLabel={'Yes, delete my account permanently!'}
-      onConfirm={handleClick}
-      mainButtonProps={{ colorScheme: 'red' }}
-    />
+    <Box onClick={handleClick} cursor="pointer">
+      <Flex align="center" ml={-2}>
+        <IconButton isRound aria-label="Delete Account" icon={<Icon as={WarningTwoIcon} />} bgColor="transparent" />
+        <Flex direction="column" ml={2}>
+          <span>Delete Account</span>
+        </Flex>
+      </Flex>
+    </Box>
```

Comment:
I think we should still have a confirmation dialog here for account deletion, kind of scary to accidentally click and delete without warning ahahah

Reply:
also the heatmap + stats looks really good, tbh i think it'd look nice on the home page too since its kinda bare (under the interview prep section), though the home page + user profile page might be kinda duplicated 🤔 ... though it might be fine if we plan to extend the implementation for user profile page to be viewable by any user (like a github profile page), which seems to be quite easy(?) to do (not 100% sure) since everything's already nicely abstracted out, we just need to abstract the userid out as a param (again, low priority)

Reply:
> I think we should still have a confirmation dialog here for account deletion, kind of scary to accidentally click and delete without warning ahahah

Agreed, i think the previous button is better now that i look at it again